cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -870,6 +870,7 @@
        ;; *execute* the embedded `Inst`. (In the emitted code, we use the inverse
        ;; of this condition in a branch that skips the trap instruction.)
        (TrapIf
+        (size OperandSize)
```

Comment:
I think we mostly had two choices here:
- either make the size part of the `CondBrKind`, which is tighter, but yields more changes in the code
- or setting the size in the op itself (current approach), which required less changes

Reply:
I think I might prefer the first option actually -- I was a bit confused at first how the `OperandSize` would affect the `TrapIf` if we're just talking about conditional branch kinds like `b.lo`, `b.hi`, etc; until I realized that `CondBrKind::Zero(reg)` and `NonZero(reg)` exist and encode `cbz`/`cbnz`. If you don't mind, do you think you could try the refactor out? On the upside, it seems like it should get rid of a bunch of extraneous `OperandSize`s on branches emitted in various places that are condcode-based (e.g. in `abi.rs` above and in many of the emit-tests below).

Reply:
Also, separately, if we modify `CondBrKind`, this is toward the direction that we want for supporting 32-bit zero/nonzero tests in regular conditional branches too. We don't have to implement that now (you can `unimplemented!()` it in the `CondBr` case; but if you wanted to, it would certainly be fine with me) but it's nice to lay the groundwork.

Reply:
This make sense to me, I'll refactor